react-exercises/modal-react/src/Components/Button/styles.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../sass/global";
+
+$btn-text-color : white;
```

Comment:
Please move this to _variables.scss file in sass folder.
also plesae re use existing color variables